www/apps/resources/app/medusa-cli/commands/build/page.mdx

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_label: "build"
+sidebar_position: 5
+---
+
+import { Table } from "docs-ui"
+
+export const metadata = {
+  title: `build Command - Medusa CLI Reference`,
+}
+
+# {metadata.title}
+
+Create a standalone build of the Medusa application. This is useful to create a build that:
+
+- Doesn't rely on the source TypeScript files.
+- Has a smaller Docker image size.
+- Can be copied to a production server reliably.
```

Comment:
thought: Don't think these add much right now, would leave them out of the docs

Reply:
What I mean is, this is not a special way to build Medusa, it is the only way, so think the guide should explain it as such